scripts/cancel_order.ts

```diff
@@ -0,0 +1,33 @@
+import { factory } from "../ConfigLog4";
```

Comment:
ubernit: I noticed this file name after the PR had already been merged, I'm not a huge fan of it. I would rather it be `src/logging.ts` and not in the root directory.

Reply:
Yea cool! I am still unfamiliar with the file structure of a TS project and will change it here. I assume this is not meant to be exported.